### PR TITLE
task_DEV-1

### PR DESCRIPTION
**Pull request by NikaMakovik**
Nika Makovik 
task_DEV-1

### CODE REVIEW

File: DEV-1/DEV-1/IdenticalSymbols.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+{
+    /// <summary>
+    /// This class contains methods to calculate number of serial identical symbols.
+    /// </summary>
+    class SerialIdenticalSymbols
+    {
+        /// <summary>
+        /// This class calculates number of serial identical symbols.
+        /// </summary>
+        /// <param name="theString"></param>
```
Comment (AlexanderSoroka) [inline DEV-1/DEV-1/IdenticalSymbols.cs]: missed param description

File: DEV-1/DEV-1/IdenticalSymbols.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        /// <returns>
+        /// Returns maximum number of serial identical symbols.
+        /// </returns>
+        static public int GetMaxNumberOfSerialIdenticalSymbols(string theString)
```
Comment (AlexanderSoroka) [inline DEV-1/DEV-1/IdenticalSymbols.cs]: There is no reason to make this method static

File: DEV-1/DEV-1/IdenticalSymbols.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        {
+            int counter = 1;
+            int maxEquals = 1;
+            for (int i = 0; i < theString.Length - 1; i++)
+            { 
+                if (theString[i + 1] == theString[i])
+                {
+                    counter++;
+                    if (counter > maxEquals)
```
Comment (AlexanderSoroka) [inline DEV-1/DEV-1/IdenticalSymbols.cs]: codestyle violation

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: DEV-1/DEV-1/EntryPoint.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+{
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length != 1)
+            {
+                Console.Write("Wrong number of arguments.");
+            }
+            else
```
Comment (AlexanderSoroka) [inline DEV-1/DEV-1/EntryPoint.cs]: useless else, just add return to if statement

File: DEV-1/DEV-1/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Общие сведения об этой сборке предоставляются следующим набором
```
Comment (AlexanderSoroka) [inline DEV-1/DEV-1/Properties/AssemblyInfo.cs]: Do not use russian

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: DEV-1/DEV-1/SerialIdenticalSymbols.cs [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    class SerialIdenticalSymbols
+    {
+        /// <summary>
+        /// This class calculates number of serial identical symbols.
+        /// </summary>
+        /// <param name="theString">The main string.</param>
+        /// <returns>
+        /// Returns maximum number of serial identical symbols.
+        /// </returns>
+        public int GetMaxNumberOfSerialIdenticalSymbols(string theString)
```
Comment (AlexanderSoroka) [inline DEV-1/DEV-1/SerialIdenticalSymbols.cs:15]: what if theString is empty? I suppose your implementation returns 1 that is wrong

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: DEV-1/DEV-1/SerialIdenticalSymbols.cs (hunk 6 above)
Comment (NikaMakovik) [reply DEV-1/DEV-1/SerialIdenticalSymbols.cs:15]: in case empty theString my implementation returns "Wrong number of arguments." as it supposed to be

Comment (AlexanderSoroka) [reply DEV-1/DEV-1/SerialIdenticalSymbols.cs:15]: Obviously this method never returns "Wrong number of arguments." as it can return only int

**Review by AlexanderSoroka — APPROVED** [review]
Approved due to deadline

**Pull request merged**